src/mariadb/mod.rs

```diff
@@ -69,6 +79,31 @@ mod tests {
         testcontainers::{runners::SyncRunner, ImageExt},
     };
 
+    #[test]
+    fn mariadb_with_init_sql() -> Result<(), Box<dyn std::error::Error + 'static>> {
+        use crate::InitSql;
```

Comment:
@DDtKey the choice of making this be a trait has the effect that this has to be imported.

That is a bit aqward imo => are you sure that you prefer it this way?

Reply:
Usually it's fine to import traits in Rust. Many extensions work this way.
The idea of trait here was to unify this modules and avoid the possible differences, while have a documentation in one place.


However, we have only several modules with such functionality and I'd say: let's just keep separate methods for each one. I mean let's go without trait for now